src_assets/common/assets/web/pin.html

```diff
@@ -8,10 +8,11 @@
 <body id="app" v-cloak>
   <Navbar></Navbar>
   <div id="content" class="container">
-    <h1 class="my-4">{{ $t('pin.pin_pairing') }}</h1>
+    <h1 class="my-4 text-center">{{ $t('pin.pin_pairing') }}</h1>
     <form action="" class="form d-flex flex-column align-items-center" id="form">
```

Comment:
With the change proposed below, use this:

```suggestion
    <form class="form d-flex flex-column align-items-center" id="form" @submit.prevent="registerDevice">
```